UITests/UITests.App/MainTestHost.xaml.cs

```diff
@@ -114,5 +114,12 @@ private void NavigationFrame_NavigationFailed(object sender, Windows.UI.Xaml.Nav
             Log.Error("Failed to navigate to page {0}", e.SourcePageType.FullName);
             _loadingStateTask.SetResult(false);
         }
+
+        private void GoBackInvokerButton_Click(object sender, RoutedEventArgs e)
+        {
+            Log.Comment("Go Back Clicked. Navigating to Page...");
+            navigationFrame.Navigate(typeof(Page));
```

Comment:
Will be cool to see how this works in the pipeline, but is this just navigating to a blank page? I think we'll be queueing up a stack of pages then with each test?

Do we want to seed our frame with a blank page at the start and `GoBack` here instead or do we want to try and disabling caching with `CacheMode` and `CacheSize` on the frame itself so we just always move forward?